mam/api/api.c

```diff
@@ -288,6 +292,26 @@ retcode_t mam_api_bundle_write_packet(
   return RC_OK;
 }
 
+retcode_t mam_api_num_remaining_sks(mam_api_t *const api,
```

Comment:
Should this be done on a channel/endpoint instead ?

Reply:
it could, and it's indeed better